app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { Engine, Render, World, Bodies } from 'matter-js';
+import { graphql } from 'react-apollo';
+import { pathOr, keys, compose } from 'ramda';
+import { getEventCounts } from './queries';
+import githubPull from '../../../../assets/images/pr.png';
+import githubCommit from '../../../../assets/images/commit.png';
+import slackMessage from '../../../../assets/images/slack.png';
+
+const blockTypes = { githubPull, githubCommit, slackMessage };
+
+class Scene extends React.Component {
```

Comment:
i'm thinking some  comments would help this component a bit as to how it will be used when implementing no? even if it were just some links to the documentation as to what some of these matter-js imports are doing.

Reply:
i don't see any propTypes on this component, is that on purpose?

Reply:
Yes, I avoid adding propTypes unless the linter complains.

Reply:
Or name them!